.github/workflows/push_images.yml

```diff
@@ -0,0 +1,109 @@
+#    Again, the test.yml workflow must be complete.
+# 3. By commenting on the PR `please update screenshots`.
+#
+# If a PR has already been opened on the conda-store repo from this
+# action, a rerun of this action will attempt to update the PR, not
+# create a new one. 
+
+name: Push screenshots to conda-store docs
+
+on:
+  pull_request:
+    types: [labeled]
+  issue_comment:
+    types: [created, edited]
+  workflow_dispatch:
+    inputs:
+      pr_number:
+        description: "PR Number with screenshots to push"
+        required: true
+
+jobs:
+  update_screenshots:
+    name: "update_screenshots"
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        shell: bash -el {0}
+    if: ${{ github.event.label.name == 'update_screenshots' && github.event_name == 'pull_request' || github.event_name == 'workflow_dispatch' || (github.event.issue.pull_request && contains(github.event.comment.body, 'please update screenshots')) }}
+    steps:
+      - name: "Checkout the conda-store repository 🛎"
+        uses: actions/checkout@v4
+        with:
+          path: conda-store-repo
+          repository: 'conda-incubator/conda-store'
+          
+      - name: Use this PR number or the one specified in workflow dispatch
+        run: |
+          if [[ ${{ github.event_name == 'workflow_dispatch' }} == true ]]; then
+            PR_NUMBER=${{ inputs.pr_number }}
+          else
+            PR_NUMBER=${{github.event.pull_request.number}}
+          fi
+            echo "PR_NUMBER=$PR_NUMBER" >> $GITHUB_ENV
+          
+      - name: Download artifact images - will fail if artifact doesn't exist yet
+        id: download-artifact
+        uses: dawidd6/action-download-artifact@v3
+        with:
+            #   Optional, GitHub token, a Personal Access Token with `public_repo` scope if needed
+            # Required, if the artifact is from a different repo
+            # Required, if the repo is private a Personal Access Token with `repo` scope is needed or GitHub token in a job where the permissions `action` scope set to `read`
+            github_token: ${{ secrets.GITHUB_TOKEN }}
+            # workflow file name from which to get the artifact
+            workflow: test.yml
+            # PR from which to get the artifact
+            pr: ${{ env.PR_NUMBER }}
+            # how to exit the action if no artifact is found
+            if_no_artifact_found: fail
+            # artifact name
+            name: 'screenshots'
+            # directory in which to put the contents of the artifact
+            path: 'screenshots'
+
+            
+      - name: "Copy images to conda-store repo"
+        run: |
+            cd conda-store-repo
+            cp ../screenshots/*.png docusaurus-docs/conda-store-ui/images/.
+            git config user.name ${{ github.actor }}
+            git config user.email '${{ github.actor }}@users.noreply.github.com'
+            git add -A
+            git commit -m "commit all changes"
```

Comment:
I would still like to see this commit message updated

```suggestion
            git commit -m "🤖 conda-store-ui/update_screenshots"
```

Reply:
ah. Sorry, I changed [this line](https://github.com/conda-incubator/conda-store-ui/pull/346/files#diff-cf07fe50afffc4f6a5fcf111db3b52acf529796eedc2b4fb65fe755cb4f029baR92) instead when you requested the first time. 

Ok... little bit of a wild goose chase, but I think I got it. The commit-message below (line 92) is being ignored and THIS line is what's being used. However, updates to the PR are just force pushing commits. Which is not updating the commit message. Therefore, I'll make this change in the right place, as you suggest, but I'll have to close the AUTO PR in order to see the change.